### PR TITLE
enable venv path for sensu on rhel

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**
on sensu-service start, it execute below  in squence :
1. . /etc/default/sensu
2. . /etc/init.d/functions (only on rhel)
3. ./etc/sysconfig/sensu-client (only on rhel)

. /etc/init.d/functions reset the $PATH, which cause PATH in
/etc/default/sensu not work.
so need to setup path in /etc/sysconfig/sensu-client.

### CODE REVIEW

**Comment by nirajdp76** [comment]
retest

**Comment by Xiaohua-Shen** [comment]
add one more commit to fix ceph related check, which avoid ceph related check to use base_venv

**Comment by Xiaohua-Shen** [comment]
retest

File: roles/common/templates/monitoring/sensu-client-path [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% if "ceph_monitors" not in group_names -%}
+export PATH="{{ common.python.base_venv }}/bin:/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
+{% else %}
+export PATH="/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
```
Comment (paulczar) [inline roles/common/templates/monitoring/sensu-client-path]: if we do this ...  it means any checks on ceph_monitor nodes would use system python.   We should install ceph pips into base_venv and keep it consistent ?

Comment (Xiaohua-Shen) [inline roles/common/templates/monitoring/sensu-client-path]: No ceph pips existed, so I used this workaround

Comment (Xiaohua-Shen) [inline roles/common/templates/monitoring/sensu-client-path]: mimic what cinder and glance did to create link for rados and rdo packages.
find another package missing:
(base)[root@cpm1 site-packages]# ceph health
Traceback (most recent call last):
  File "/bin/ceph", line 124, in <module>
    from ceph_argparse import \
ImportError: No module named ceph_argparse

**Pull request merged**